packages/teleport/src/Roles/templates/role.yaml

```diff
@@ -6,20 +6,55 @@ spec:
   # This example defines an administrative role. It maps to Kubernetes "admin"
   # group and allows SSH to every node.
   allow:
+    # List of Kubernetes cluster users can access the k8s API
+    kubernetes_labels:
+      '*': '*'
     # This role is mapped to Kubernetes 'admin' group.
     kubernetes_groups: [admin]
-
+    
     # List of allowed SSH logins
-    logins: [root]
+    logins: [ubuntu, debian]
 
     # List of node labels that users can SSH into
     node_labels:
       '*': '*'
+      
+    # List of application labels users can access
+    app_labels:    
+      '*': '*'
+      
+    # List of database labels users can access database servers
+    db_labels:
+      '*': '*'
+    # List of databases on the database server users can access
+    db_names:
+     - '*'
+    # List of database users allowed to open database connections with
+    db_users:
+    - dba-admin
+    
+    # List of windows desktop access labels that users can open desktop sessions to
+    windows_desktop_labels:
+      '*': '*'
+    # Windows logins a user is allowed to use for desktop sessions.
+    windows_desktop_logins:
+    - '{{internal.windows_logins}}'
+    - Administrator
```

Comment:
If we didn't make `root` a default Unix login, then we probably shouldn't make `Administrator` a default Windows login.

Reply:
I changed to use developer for that and other examples